packages/cli-repl/src/completer.spec.ts

```diff
@@ -0,0 +1,125 @@
+import write from './completer';
```

Comment:
is `write` as name intended, isn't something like `completer` or `complete` more clear?

Reply:
ha! i copied this over from another test and forgot to rename